components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -231,6 +231,8 @@ lnd/clm2/surfdata_map/surfdata_ne0np4_northamericax4v1.pg2_simyr2010_c210112.nc<
 lnd/clm2/surfdata_map/surfdata_arcticx4v1pg2_simyr2010_c210805.nc</fsurdat>
 <fsurdat hgrid="ne0np4_arcticx4v1.pg2"   sim_year="2000" use_crop=".false." >
 lnd/clm2/surfdata_map/surfdata_arcticx4v1pg2_simyr2000_c210707.nc</fsurdat>
+<fsurdat hgrid="ne0np4_arcticx4v1.pg2"   sim_year="1950" use_crop=".false." >
+lnd/clm2/surfdata_map/surfdata_arcticx4v1pg2_simyr2000_c210707.nc</fsurdat>
```

Comment:
I've also got the `finidat` spunup file ready and staged.

Reply:
^actually I see you're pointing to the 2000 year file, and we have a new 1950 one we should use here (h/t @brhillman)

Reply:
HI @darincomeau.  This is really just a place holder.  It turns out, if fsurfdat is not set to something by default then case.setup fails.  Separately, we generally set fsurfdat in our user_nl_files, so the default is overwritten.  It's possible that you haven't noticed this problem for a number of reasons.  It doesn't affect the case setup because it just causes preview_namelist to fail until user_nl_elm is placed in the case directory.  And that is the last step of case.setup.  Maybe also because you run the case setup script and don't notice some of the output.  Anyway, this just fixes a case setup problem but is not meant to be a long term setting.

It's great that you have an finidat file and may even have a better fsurfdat file.  And I welcome those being set in the scripts as defaults.

Reply:
@darincomeau: you pointed me to this new `finidat` file: `/lcrc/group/e3sm/data/inputdata/lnd/clm2/initdata/20221017.ICRUELM-1950.arcticx4v1pg2_ARRM10to60E2r1.anvil.elm.r.0051-01-01-00000.nc`
Could we use this one in this PR and then merge?

Reply:
that means that I will start the new simulations right after step 3 and changing finidat  in my user_nl file.